languageserver/server/server.go

```diff
@@ -1788,27 +1788,24 @@ func (s *Server) getDiagnostics(
 
 					importedLocationID := importedLocation.ID()
 
-					importedChecker, ok := s.checkers[importedLocationID]
```

Comment:
👍 

Maybe add a comment explaining that we re-check all imports for the moment and do not cache imported checkers, because we do not implement proper dependency tracking yet which would be necessary for proper cache invalidation. In addition, maybe also link to the new issue